tdrs-backend/tdpservice/users/api/authorization_check.py

```diff
@@ -65,9 +65,29 @@ def get(self, request, *args, **kwargs):
 
         user_in_valid_group = user.is_ofa_sys_admin or user.is_digit_team
 
-        if (user.hhs_id is not None and user_in_valid_group) or settings.BYPASS_KIBANA_AUTH:
+        if (user.hhs_id is not None and user_in_valid_group) or settings.BYPASS_OFA_AUTH:
             logger.debug(f"User: {user} has correct authentication credentials. Allowing access to Kibana.")
             return HttpResponse(status=200)
         else:
             logger.debug(f"User: {user} has incorrect authentication credentials. Not allowing access to Kibana.")
             return HttpResponse(status=401)
+
+class GrafanaAuthorizationCheck(APIView):
+    """Check if user is authorized to view Grafana."""
+
+    query_string = False
+    pattern_name = "grafana-authorization-check"
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request, *args, **kwargs):
+        """Handle get request and verify user is authorized to access grafana."""
+        user = request.user
+
+        user_in_valid_group = user.is_ofa_sys_admin or user.is_developer
```

Comment:
Since we plan to only have a single grafana instance, we can access it in the envs where the Developer group is defined/used. We dont need to be in prod to see it